Makefile.in

```diff
@@ -1728,6 +1728,7 @@ echo-list	= for x in $1; do $(ECHO) "$$x"; done
 
 .PHONY: all
 all: $(default_pdf_targets) ;
+	killall -q -SIGHUP mupdf || true
```

Comment:
Wow, yes, that *is* a dirty hack (and it won't work on Mac because it doesn't have killall, and Windows doesn't either, obviously).

My suggestion for hacks like these is that we document them somewhere and make them available as a setting in a Makefile.ini.

Have you tried putting something like this into your Makefile.ini to see if it works without editing the Makefile itself?

all:
  killall -q -SIGHUP mupdf || true

That *should* simply add a dependency to all, instead of overwriting it. Then we can keep these kinds of specific hacks in the .ini files where they are ideally supposed to be. Ideals don't always work out, though, so I'm curious to see how this works for you.